lte/cloud/go/services/subscriberdb/load.go

```diff
@@ -231,3 +231,25 @@ func ConvertSubEntsToProtos(ent configurator.NetworkEntity, apnConfigs map[strin
 
 	return subData, nil
 }
+
+func LoadSuciProtosPage(ctx context.Context, networkID string) ([]*lte_protos.SuciProfile, error) {
```

Comment:
We're really loading all of them, not just a single page

```suggestion
func LoadSuciProtos(ctx context.Context, networkID string) ([]*lte_protos.SuciProfile, error) {
```

Reply:
done